webapp/packages/core-blocks/src/FormControls/isControlPresented.ts

```diff
@@ -6,9 +6,14 @@
  * you may not use this file except in compliance with the License.
  */
 
-export function isControlPresented(name: string | undefined, state: any): boolean {
+export function isControlPresented(
+  name: string | undefined, state: any, defaultValue?: string | number | boolean
```

Comment:
defaultValue can't be boolean, it can be `string | number | ReadonlyArray<string>`